Technical-Steering-Committee/issues.md

```diff
@@ -23,27 +23,46 @@ A question about how something works in the specific projects. See the [recommen
 - **bug**  
 A defect found in one of the existing project functionalities. See the [recommended template](https://github.com/zowe/api-layer/blob/v2.x.x/.github/ISSUE_TEMPLATE/bug_report.md).
 
-Importance of the issue as perceived by squad organized by importance:
+## Triaging Guidelines: *Bugs*
+
+Importance of the issue as perceived by squad:
 
 - **priority-critical**  
-An issue that needs to be resolved within this PI, ideally within next sprint
+*Address Immediately*: The submitter has indicated (or the reviewer has judged) that the defect is causing significant detrimental impact such as production outage, data corruption, security exposure, etc. The issue should be pulled in immediately and resolved. If this is not possible, the issue should be escalated to the TSC immediately.
```

Comment:
Do we conflate priority and severity here? I don't think an issue needs to cause any of these to still be "priority-critical". Also, should the submitter assign a priority?

Reply:
Regarding these concrete examples, I see below that now there are separate sections for bugs and enchantments, and that's why all these details need to be provided.